co-circom/circom-mpc-vm/src/mpc.rs

```diff
@@ -117,4 +117,14 @@ pub trait VmCircomWitnessExtension<F: PrimeField> {
 
     /// num2bits accelerator
     fn num2bits(&mut self, a: Self::VmType, bits: usize) -> Result<Vec<Self::VmType>>;
+
+    /// addbits accelerator
+    ///
+    /// Returns the binary addition result of two bit vectors a and b. The bit vectors are expected to have the same length. Also returns the carry output bit.
+    /// a and b are ordered such that the MSB is at the first index, as well as the result
```

Comment:
I think everywhere else we have the first index to be the LSB. What is the design choice here for this?

Reply:
That is the convention that Reclaim is using internally, so I did not want to change it.
Another issue is probably that we do not want to have the accelerators in this trait?